src/index.js

```diff
@@ -178,15 +178,13 @@ export default async function fetch(url, options_) {
 			}
 
 			// Prepare response
-			response_.once('end', () => {
-				if (signal) {
+			if (signal) {
```

Comment:
note: `signal` here is declared with `const` and thus cannot change from null to object or vice versa

Reply:
> note: `signal` here is declared with `const` and thus cannot change from null to object or vice versa